cpp/dolfinx/mesh/MeshTags.h

```diff
@@ -76,6 +76,32 @@ class MeshTags
   /// Move assignment
   MeshTags& operator=(MeshTags&& tags) = default;
 
+  /// Find all MeshTags for a list of values and return the corresponding
+  /// indices per value, local-to-process
+  /// @param[in] values A list values
+  /// @return The corresponding indices
+  std::vector<std::vector<std::int32_t>>
```

Comment:
Is this a helpful return type given that `locate_dofs_topological` takes an Eigen array?